modules/microshift-greenboot-check-status.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/checking-greenboot-scripts-status.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-check-status_{context}"]
+= Checking the status of Greenboot health checks
+
+Check the status of Greenboot health checks before making changes to the system or during troubleshooting. You can use any of the following several commands to help you ensure that Greenboot scripts have finished running.
+
+.Procedure
+
+* To see a report of health check status, use the following command:
++
+[source,terminal]
+----
+$ systemctl show --property=SubState --value greenboot-healthcheck.service
+----
++
+** An output of "running" means that Greenboot checks are still running.
+** An output of "success" means that checks have passed and Greenboot has exited.
+** An output of "failed" means that checks have not passed. Greenboot runs the checks in `red.d` when the system is in this state and might restart the system.
```

Comment:
This is kind of another way around, systemd will print these values: `start`, `exited`, `failed` which we need to interpret in following way:

- `start` means healthchecks are in progress, running
- `exited` means that healthchecks ended with success
- `failed` means that healthchecks ended with failure

Reply:
You included info that "greenboot runs checks in red.d" on failed:

1. `red.d` does not include checks, these script don't check anything anymore, only act in case of greenboot failure
2. if we included that `failed` -> run `red.d`, then we might also included that `exited` -> run `green.d`